### ISSUE TITLE
Sonos binding: problem with mute channel

### ISSUE THREAD

**Issue by lolodomo** [eclipse/smarthome#1900]
Assuming your Sonos sound is not mutted, you use the switch associated to the "mute" channel in UI to mute it. That works but immediately after the switch state is restored to OFF state. Important, I am using a switch with autoupdate=false.

Tested with beta 3.

Just in case it could help: if I correctly remember, there is something weird with mute in the Sonos firmware:  you have to ignore case because sometimes you get "mute" rather than "Mute" or the reverse.

**Comment by lolodomo** [comment]
I investigated this issue and discovered what's wrong.
In methoid onValueReceived, the value is compared to "On" while the received value is either "0" or "1". As a result, "1" leads to set the channel to OFF state.
```
                case "MuteMaster": {
                    logger.debug("onValueReceived MuteMaster {}", stateMap.get("MuteMaster"));
                    State newState = UnDefType.UNDEF;
                    if (stateMap.get("MuteMaster") != null) {
                        if (stateMap.get("MuteMaster").equals("On")) {
                            newState = OnOffType.ON;
                        } else {
                            newState = OnOffType.OFF;
                        }
                    }
                    updateState(new ChannelUID(getThing().getThingTypeUID(), getThing().getUID(), MUTE), newState);
                    break;
                }
```
Why checking "On" value ? Do I miss something ?
This comparison with "On" is done for several variables: CurrentLEDState, GroupCoordinatorIsLocal, MuteMaster, LineInConnected, AlarmRunning,

**Comment by kgoderis** [comment]
This could very well be due to a change in the Sonos firmware. The binding was developed a number of firmware iterations ago and I do not test it fully against each firmware update. Any ways you found the issue and it only requires a quick fix

**Comment by lolodomo** [comment]
I let you propose a fix ?
Please check all the channels where there is this comparison with "On".

**Comment by lolodomo** [comment]
I will propose the fix.

**Comment by kaikreuzer** [comment]
fixed by #1905
